packages/dotcom/src/requests.ts

```diff
@@ -0,0 +1,71 @@
+import { BannerPayload, EpicPayload, HeaderPayload } from '../../shared/src/types/payloads';
+import { TestTracking } from '../../shared/src/types/tests/shared';
+
```

Comment:
the helpers in this file are new, though based on the existing fetching logic in DCR